File: saml/src/main/java/com/zorroa/security/saml/ZorroaSAMLAuthenticationProvider.java
package com.zorroa.security.saml;

import org.springframework.security.saml.SAMLAuthenticationProvider;
import org.springframework.security.saml.SAMLCredential;

import java.util.Date;

public class ZorroaSAMLAuthenticationProvider extends SAMLAuthenticationProvider {

    @Override
    protected Date getExpirationDate(SAMLCredential credential) {
        return null;
    }
}

File: saml/src/main/kotlin/com/zorroa/security/saml/ZorroaSAMLAuthenticationProvider.kt
package com.zorroa.security.saml

import org.springframework.security.saml.SAMLAuthenticationProvider
import org.springframework.security.saml.SAMLCredential

import java.util.Date

class ZorroaSAMLAuthenticationProvider : SAMLAuthenticationProvider() {

    override fun getExpirationDate(credential: SAMLCredential): Date? {
        return null
    }
}
